spring-integration-core/src/main/java/org/springframework/integration/annotation/Poller.java

```diff
@@ -25,7 +25,7 @@
  * for the Messaging annotations for polled endpoints. It is an analogue of the XML
  * {@code <poller/>} element, but provides only simple attributes. If the
  * {@link org.springframework.integration.scheduling.PollerMetadata} requires more options
- * (e.g. Transactional and other Advices) or {@code initialDelay, receiveTimeout} etc, the
+ * (e.g. Transactional and other Advices) or {@code initialDelay} etc, the
```

Comment:
Good catch!

Would you mind to fix other typos in this JavaDoc then?

- `etc` should come with `.`
- there is a double `the` in the `errorChannel`
- `whether or not` must be without `or not`

All the stuff my IDEA suggest to me 😄